src/channels/07-explorables.yml

```
@@ -7,6 +7,9 @@ schedule:
         author: Nicky Case
         url: https://ncase.me/sim/
     - "00:30":
+        title: FORTUNE.EXE
+        author: Ｐａｃｋａｒｄ Ｂｅｌｌ ９５ メモリー
```

```suggestion
        author: Packard Bell 95
```
Credits font only includes the latin-ext character subset so we'll need to change this

---

Makes sense. Thanks for pointing this out. I've adjusted the author